server/migrations/20210720192451-create-assets-visualization.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
```

Comment:
I think this file is un-intentionally added here as it was deleted from Master branch